support/debug/recovery/panic_handler.go

```diff
@@ -0,0 +1,17 @@
+package recovery
+
+import (
+	"github.com/goravel/framework/contracts/support/debug/recovery"
+)
+
+type PanicHandler struct{}
```

Comment:
Q: Can you please give an example for how to use this in goravel/goravel?

Reply:
It involves my another discussion: https://github.com/goravel/goravel/issues/209

My idea is to have a Kernel to start the entire application, and this startup is a root command.

I can configure the recovery from this root command.

Reply:
Okay, let's discuss it at https://github.com/goravel/goravel/issues/209